client/integration-tests/serializable-model/src/test/java/io/quarkiverse/openapi/generator/it/QuarkusSerializableModelRestEasyReactiveTest.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkiverse.openapi.generator.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.Serializable;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+@Tag("resteasy-reactive")
```

Comment:
You only need one file for each RESTEasy implementation when they have different tests, which is not the case.
So, you can keep one single file and remove `@Tag`. The CI will be in charge of testing both implementations using profiles.

For more info: https://github.com/quarkiverse/quarkus-openapi-generator/blob/main/CONTRIBUTING.md#resteasy-reactive-and-resteasy-classic